service/openapi.yaml

```diff
@@ -279,3 +314,33 @@ components:
     errorDetails:
       type: string
       description: An error description for logging or other needs
+    metadataResponse:
+      description: OFREP metadata response
+      properties:
+        name:
+          type: string
+          description: name of the flag management system
+          examples:
+            - flagd
+            - go-feature-flag
+        features:
+          type: object
+          description: List of the features supported by the flag management system.
+          properties:
+            cacheInvalidation:
+              type: array
+              enum: [POLLING]
```

Comment:
Each of these `cacheInvalidation` types would need its own documentation. Where do we keep those?

Reply:
For now we only have the support of 1 type of cache invalidation (aka polling), this is more here to extendable in the future.